wpeverest/user-registration
# Refactor - Login form processing

### All Submissions:






### Changes proposed in this Pull Request:


This PR refactors the code for login form processing.

### How to test the changes in this Pull Request:

1. Verify, whether the login (both AJAX as well as without ajax) is working or not.

### Types of changes:




### Other information:




### Changelog entry

> Refactor - Login form processing.
